tokens/umint.json

```diff
@@ -0,0 +1,10 @@
+{
+  "ethereum_address": "0xE99De844EF3Ef72806cf006224EF3b813e82662f",
+  "aurora_address": "",
```

Comment:
Deployed an Aurora token at: `0x984c2505a14da732d7271416356f535953610340`. Please add to the PR.

Reply:
Done, thanks!